Update README.md

andpeterson opened the pull request:
Added a guide, needs to be more fleshed out.
-Setup
-Using SSH
-Known Issues
-Code Structure

andpeterson committed:
Update README.md

Added a guide, needs to be more fleshed out.
-Setup
-Using SSH
-Known Issues
-Code Structure

amberhu-uw commented:
Thanks, Andrew. A couple things that I want to get done before we merge (if I have time I will probably do them myself)

- ~~Add full Pi4J Hardware issue error code~~
- ~~Define mount in RPi Setup~~
- ~~Flesh out code structure~~
- Add instructions to change the username for the main user as well as the system name for the different berries 
- ~~Add UWIGEM image at the top for style~~
- ~~Link the Drylab Trello board~~
- ~~Add the NetBeans subsection, and add it to the Raspberry Pi Setup subsection~~
- ~~Change the Raspberry Pi Setup subsection into just the Setup section~~
- ~~Flesh out the Using SSH section~~

amberhu-uw committed:
Added UWIGEM Logo

Replaced title header

amberhu-uw committed:
Added Trello link

Added it to the TODO section at the bottom. Also moved the README example link to the very bottom

amberhu-uw committed:
Added full Pi4J hardware error dialog

amberhu-uw committed:
Clarified 'mount' in Setup

Change ISO to IMG, since Raspbian uses a .img instead of a .iso

amberhu-uw committed:
+ Added NetBeans setup subsection

amberhu-uw committed:
= Fixed UWIGEM logo not displaying

Forgot that I had moved the image into an img folder

amberhu-uw committed:
= Fixed lack of newline

between the paragraphs under #TODO

amberhu-uw committed:
+ Fleshed out Code Structure & added tip

Added tip about closing projects instead of deleting them under Installing NetBeans

amberhu-uw committed:
+ Fleshed out Using SSH section

Updated TODO list

amberhu-uw commented:
Alright, it looks like the README is ready to go for now. We will make sure to keep updating it as we go.

One thing we do still need to do:

- Add instruction to change the username for the main user as well as the system name for the different berries during setup

I didn't add this in yet because I still need to talk to Jase about whether this will be necessary at all or not
